swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = None, condition = 0, age = 0):
+        if not category:
```

Comment:
If we don't specify a category, it will be `None` because of `category = None` in the constructor!